src/components/menu/examples/menu-grid.tsx

```diff
@@ -3,15 +3,16 @@ import { Component, h } from '@stencil/core';
 
 /**
  * With grid layout
- * To render items of a menu in a gird layout instead of a vertical list,
- * two things are required:
- * 1. Simply setting the `gridLayout` property to `true`, and
- * 1. Specifying a proper width for the menu surface, using the
- * `--menu-surface-width` variable.
+ * To render items of a menu in a grid layout instead of a vertical list,
+ * simply setting the `gridLayout` property to `true`.
  *
  * :::note
- * Do not forget that menus should behave responsively, thus using a fixed `width`
- * should be avoided. To make the width responsive, try using the `min()` function.
+ * The menu surface of menus with the grid layout has a responsive width by default,
```

Comment:
"The menu surface of menus" can be shortened to just "Menus":

```suggestion
 * Menus with the grid layout has a responsive width by default,
```

Although, that's true for non-grid menus too, it's just the maximum width that differs…